rpc: add context toggle to getProgramAccounts (#17399)
solana-labs/solana: docs/src/developing/clients/jsonrpc-api.md

    "jsonParsed" encoding attempts to use program-specific state parsers to return more human-readable and explicit account state data. If "jsonParsed" is requested but a parser cannot be found, the field falls back to "base64" encoding, detectable when the `data` field is type `<string>`.
  - (optional) `dataSlice: <object>` - limit the returned account data using the provided `offset: <usize>` and `length: <usize>` fields; only available for "base58", "base64" or "base64+zstd" encodings.
  - (optional) `filters: <array>` - filter results using various [filter objects](jsonrpc-api.md#filters); account must meet all filter criteria to be included in results

>   - (optional) `withContext: bool` - wrap the returned array into RpcResponse JSON object.

Reviewer: ```suggestion
  - (optional) `withContext: bool` - wrap the result in a RpcResponse JSON object.
```
